test-framework/test-suites/integration/tests/remove/test_remove_environment_route.py

```diff
@@ -8,6 +8,11 @@ def test_remove_environment_route(self, host):
 		result = host.run('stack add environment route test address=test gateway=test')
 		assert result.rc == 0
 
+		result = host.run('stack list environment route output-format=json')
+		assert result.rc == 0
+		with open('/export/test-files/remove/remove_environment_route.json') as f:
```

Comment:
all output files in the test-files directory should be named `$FILE_output.json`